src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -127,6 +127,7 @@
     <Compile Include="System\Net\Sockets\SendPacketsElementFlags.Windows.cs" />
     <Compile Include="System\Net\Sockets\Socket.Windows.cs" />
     <Compile Include="System\Net\Sockets\SocketAsyncEventArgs.Windows.cs" />
+    <Compile Include="System\Net\Sockets\IOControlKeepAlive.cs" />
```

Comment:
This should be moved to the Windows-only section.

Reply:
It is in the `ItemGroup` with `Condition=" '$(TargetsWindows)' == 'true'"`
Should I rename the file `IOControlKeepAlive.Windows.cs`?

Reply:
Ah, I see.  Yeah, let's add .Windows to the file name.